File: Java/src/Car.java
public class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer numberOfPassengers;

    public Car(Integer id, String license, Account driver, Integer numberOfPassengers) {
        this.id = id;
        this.license = license;
        this.driver = driver;
        this.numberOfPassengers = numberOfPassengers;
    }

    @Override
    public String toString() {
        return "Car{" +
                "id=" + id +
                ", license='" + license + '\'' +
                ", driver=" + driver +
                ", numberOfPassengers=" + numberOfPassengers +
                '}';
    }
}

File: CSharp/Car.cs
using System;

namespace CSharp
{
    class Car
    {
        private int id;
        private string license;
        private Account driver;
        private int numberOfPassengers;

        public Car(int id, string license, Account driver, int numberOfPassengers)
        {
            this.id = id;
            this.license = license;
            this.driver = driver;
            this.numberOfPassengers = numberOfPassengers;
        }

        public override string ToString()
        {
            return $"ID: {id}, License: {license}, Driver: {driver}, Number of Passengers: {numberOfPassengers}";
        }
    }
}